server/src/main/java/org/apache/gravitino/server/web/rest/StatisticOperations.java

```diff
@@ -428,38 +427,32 @@ public Response dropPartitionStatistics(
     }
   }
 
+  @VisibleForTesting
+  static PartitionRange.BoundType getFromBoundType(boolean inclusive) {
+    return inclusive ? PartitionRange.BoundType.CLOSED : PartitionRange.BoundType.OPEN;
+  }
+
   private static String getFormattedFromPartitionName(
-      String fromPartitionName, boolean fromExclusive) {
-    if (fromExclusive) {
-      Preconditions.checkArgument(
-          StringUtils.isNotBlank(fromPartitionName),
-          "'from' partition name must not be null or empty when 'fromExclusive' is true");
-      return "[" + fromPartitionName;
+      String fromPartitionName, boolean fromInclusive) {
+    if (fromPartitionName == null) {
+      return "(-INF";
     } else {
-      if (fromPartitionName == null) {
-        return "(-INF";
-      } else {
+      if (fromInclusive) {
         return "(" + fromPartitionName;
```

Comment:
Should you change to "[" for `fromInclusive` to `true`?

Reply:
Fixed. Sorry for this.